opendj-packages/opendj-docker/alpine/Dockerfile

```diff
@@ -2,31 +2,43 @@ FROM openjdk:8-jre-alpine
 
 MAINTAINER Open Identity Platform Community <open-identity-platform-opendj@googlegroups.com>
 
-ARG ADD_BASE_ENTRY="--addBaseEntry"
+ENV ADD_BASE_ENTRY="--addBaseEntry" \
+    PORT=1389 \
+    LDAPS_PORT=1636 \
+    BASE_DN=${BASE_DN:-"dc=example,dc=com"} \
+    ROOT_USER_DN=${ROOT_USER_DN:-"cn=Directory Manager"} \
+    ROOT_PASSWORD=${ROOT_PASSWORD:-"password"} \
+    SECRET_VOLUME=${SECRET_VOLUME} \
+    OPENDJ_SSL_OPTIONS=${SSL_OPTIONS:-"--generateSelfSignedCertificate"} \
+    MASTER_SERVER=${MASTER_SERVER} \
+    OPENDJ_REPLICATION_TYPE=${OPENDJ_REPLICATION_TYPE} \
+    OPENDJ_USER="opendj"
 
-ARG PORT=1389
-
-ARG LDAPS_PORT=1636
-
-ARG BASE_DN="dc=example,dc=com"
+ARG VERSION=@project_version@
 
-ARG ROOT_USER_DN="cn=Directory Manager"
+WORKDIR /opt
 
-ARG ROOT_PASSWORD=password
+RUN apk add --update --no-cache --virtual builddeps wget unzip \
```

Comment:
Got error while building image
```
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c apk add --update --no-cache --virtual builddeps wget unzip  && apk upgrade --update --no-cache  && apk add bash  && wget --quiet https://github.com/OpenIdentityPlatform/OpenDJ/releases/download/$VERSION/opendj-$VERSION.zip  && unzip opendj-$VERSION.zip  && apk del builddeps  && rm -r opendj-$VERSION.zip  && addgroup -S $OPENDJ_USER  && adduser -S -u 1001 -G $OPENDJ_USER $OPENDJ_USER  && install -d -o $OPENDJ_USER /opt/opendj/data  && chown -R $OPENDJ_USER:$OPENDJ_USER /opt/opendj  && chmod -R g=u /opt/opendj]: exit code: 8
```

Reply:
You'll have to pass the buildarg VERSION (version of opendj) and build from the opendj-docker:

`docker build -t openidentityplatform/opendj:4.4.11-alpine --build-arg VERSION=4.4.11 -f alpine/Dockerfile .`

Reply:
Ok, got it, thank you for the contribution @casell !